DependencyInjection/Controllers/EmailController.cs

```diff
@@ -0,0 +1,37 @@
+﻿using DependencyInjection.EmailService;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace DependencyInjection.Controllers
+{
+    [Route("api/email")]
+    public class EmailController : Controller
+    {
+        private readonly IEmailService _emailService;
+        public EmailController(IEmailService emailService)
+        {
+            _emailService = emailService;
+
+        }
+        [HttpPost("send")]
+        public void Send(string message, string receiver)
+        {
+            Client cA = new Client(_emailService);
+            cA.SendEmail(message, receiver);
+
+            /// ====> using new 
+            /// 
+            PostmarkService r1 = new PostmarkService();
```

Comment:
when we have the service registered in the DI container, it's a bad practice to initiate it manually.